data/models/company.py

```diff
@@ -79,6 +79,17 @@ class Meta:
     matched = models.BooleanField(
         default=False, verbose_name="La Company Compl'Alim a été matchée avec un Etablissement TeleIcare"
     )
+    old_siret = models.CharField(
+        "n° SIRET dans TeleIcare",
+        help_text="14 chiffres",
+        unique=True,
+        blank=True,
+        null=True,
```

Comment:
je pense que c'est possible de faire `unique=True` et `blank=True`, mais j'hesite car [les résponses ici ](https://stackoverflow.com/questions/17257031/django-unique-null-and-blank-charfield-giving-already-exists-error-on-admin-p) sont mélangés. Est-ce que c'est possible de tester si on peut sauvegarder plusieurs entreprises avec un "" ?

Reply:
En tout cas, on n'a pas de company dont l'un des champs `vat` ou `siret` est blank.
Du coup, je pense que ce `blank=True` n'est pas nécessaire ni pour `vat` ou `siret` ni pour `old_vat` ou `old_siret`.
Et notre front ne permet pas d'indiquer un siret ou vat à "".

Après test, tu as raison, il n'est pas possible d'avoir plusieurs entreprise avec la même valeur "" dans l'un de ces champs sans ValidationError `Un objet Entreprise avec ce champ N° SIRET existe déjà.`

Donc je supprime cette contrainte f5cc85dc39daf64c6694ded90bef9a5c91e03829

Reply:
Je la remets, elle est nécessaire pour la validation de l'admin b0a23c7ce52cb7a76855b7b29af989a598de456a, j'ajoute un save custom pour empêcher toute présence de "" dans la BDD